src/main/java/com/controller/orders/OrdersController.java

```diff
@@ -0,0 +1,81 @@
+package com.controller.orders;
+
+import com.dto.order.Order;
+import com.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.propertyeditors.CustomDateEditor;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@Controller
+public class OrdersController {
```

Comment:
It is better to add _RequestMapping("/orders")_ to the class, and for all methods use following URLs:
_/editOrder -> /orders/edit_
_/addOrder -> /orders/add_
_/deleteOrder -> /orders/delete_